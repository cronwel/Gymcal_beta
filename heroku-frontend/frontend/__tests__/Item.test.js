import ItemComponent from '../components/Item';
import { shallow, mount} from 'enzyme';
import toJSON from 'enzyme-to-json';

const fakeItem = {
  id: 'asdgf',
  title:'asdf',
  price:'334',
  description:'13241 25rbdt hrtgwfgwfgwr ',
  image:'asdlf.jpg',
  largeImage:'wlgreaihbvjk.jpg',
}



describe('<Item/>', () => {
  it('Displays properly', () => {
    const wrapper = shallow(<ItemComponent item={fakeItem} />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});