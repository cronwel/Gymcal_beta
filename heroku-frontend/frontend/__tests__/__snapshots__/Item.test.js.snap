// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Item/> Displays properly 1`] = `
<ItemStyles__Item>
  <img
    alt="asdf"
    src="asdlf.jpg"
  />
  <Title>
    <Link
      href={
        Object {
          "pathname": "/item",
          "query": Object {
            "id": "asdgf",
          },
        }
      }
    >
      <a>
         
        asdf
         
      </a>
    </Link>
  </Title>
  <PriceTag>
    $3.34
  </PriceTag>
  <div
    className="buttonList"
  >
    <Link
      href={
        Object {
          "pathname": "/update",
          "query": Object {
            "id": "asdgf",
          },
        }
      }
    >
      <a>
        Edit âˆ†
      </a>
    </Link>
    <button>
      View 
    </button>
    <AddToCart
      id="asdgf"
    />
    <ItemDelete
      id="asdgf"
    >
      Delete
    </ItemDelete>
  </div>
</ItemStyles__Item>
`;
